docs/platform/developing/installing.md

```diff
@@ -2,81 +2,76 @@
 
-- Ubuntu 16.04/14.04/12.04
-- openSUSE 13.2/13.1/12.3/Leap 42.1
-- Fedora 23/22/21/20
-- CentOS 7/6
-- Debian 8/7
+- Ubuntu 20.04/18.04/16.04/14.04
+- openSUSE 13.2/Leap 42.1/42.3
+
+> [!NOTE]
+>
+> Tizen SCM tools are maintained in http://download.tizen.org/tools/latest-release/ .
+> If you cannot find your Linux system, that means currently Tizen SCM tools doesn't support your operating system.
+> But, if you still want to use tools,
+> you can find in http://download.tizen.org/tools/archive/, Tizen SCM tools are stored in that repository.
 
 You can install a variety of development tools, including:
 
 - **Git Build System (GBS)** command line tool that supports Tizen package building.
 - **Image Creator (MIC)** command line tool that supports Tizen image creation.
 
-> **Note**
+> [!NOTE]
 >
 > GBS and MIC are used as examples because they are mandatory development tools for Tizen developers.
 
 ## Installing Development Tools in Ubuntu or Debian
 
 To install a development tool in Ubuntu or Debian:
 
-> **Note**
+> [!NOTE]
 >
 > The `apt-get install <Package_Name>` command is recommended because it upgrades 1 or more already installed packages without upgrading every package installed, whereas the `apt-get upgrade` command installs the newest version of all currently installed packages. In addition, `apt-get update` must always be executed before `apt-get install <Package_Name>` or `apt-get upgrade`, to resynchronize the package index files.
 
-1. Open the package manager source list using a text editor.
-
-   VIM is used in the following example:
+1. Use the following command to set up the Tizen repository in your apt list:
 
    ```
-   $ sudo vim /etc/apt/sources.list
+   echo "deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_$(lsb_release -rs)/ /" | \
+   sudo tee /etc/apt/sources.list.d/tizen.list > /dev/null
    ```
+   > [!NOTE]
+   >
+   > The `lsb_release -rs` sub-command returns the version of your Ubuntu distribution.
 
-2. Add the Tizen tools repository to the source list.
-
-   For example:
-
-   - In Ubuntu 16.04, append the following line to the source list:
-
-     ```
-     deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_16.04/ /
-     ```
-
-   - In Ubuntu 14.04, append the following line to the source list:
-
-     ```
-     deb http://download.tizen.org/tools/latest-release/Ubuntu_14.04/ /
-     ```
 
-   > **Note**
+   > [!NOTE]
+   >
+   > You can set up the Tizen repository manually using following steps:
+   > 1. Open the package manager source list using a text editor:
+   >
+   >    VIM is used in the following example:
+   >
+   >    ```
+   >    $ sudo vim /etc/apt/sources.list.d/tizen.list
+   >    ```
+   >
+   > 2. Add the Tizen tools repository to the source list:
+   >
+   >    ```
+   >    deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_<version>/ /
+   >    ```
+   >
+   >    For example:
```

Comment:
Please explain more detail?
I already insert example of Ubuntu 20.04 like below
   >    For example:
   >
   >    - In Ubuntu 20.04, append the following line to the source list:
   >
   >      ```
   >      deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_20.04/ /
   >      ```
How I append example?

Reply:
@jhyuni, since there is only one example case, @mijin85cho suggested to merge contents in line 62 with line 60. Kindly update line 60 as mentioned below and delete line 62. Hope this clarifies now.

Line 60 should now be - For example in Ubuntu 20.04, append the following line to the source list: